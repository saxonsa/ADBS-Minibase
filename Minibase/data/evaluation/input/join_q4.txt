Q(x, t) :- R(x, y, z), S(x, w, t)
