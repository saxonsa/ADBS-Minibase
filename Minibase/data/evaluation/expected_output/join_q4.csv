1, 8
2, 2
4, 3
5, 4
8, 5
